types/vue-apollo.d.ts

```diff
@@ -21,7 +21,7 @@ type VueApolloOptions = {
 export class VueApollo implements PluginObject<{}> {
   [key: string]: any;
   install: PluginFunction<{}>;
-  constructor (options: { defaultClient: ApolloClient<{}>, defaultOptions?: VueApolloOptions });
+  constructor (options: { defaultClient: ApolloClient<{}>, defaultOptions?: VueApolloOptions, clients: { [key: string]:any } });
```

Comment:
Ahhh yes I should run test and thank you for your suggestions! I am getting myself familiar with TypeScript still, your TS file saved my day 👏